google/resource_logging_project_exclusion.go

```diff
@@ -0,0 +1,83 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceLoggingProjectExclusion() *schema.Resource {
+	schm := &schema.Resource{
+		Create: resourceLoggingProjectExclusionCreate,
+		Read:   resourceLoggingProjectExclusionRead,
+		Delete: resourceLoggingProjectExclusionDelete,
+		Update: resourceLoggingProjectExclusionUpdate,
+		Schema: resourceLoggingExclusionSchema(),
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+	}
+	schm.Schema["project"] = &schema.Schema{
+		Type:     schema.TypeString,
+		Optional: true,
+		ForceNew: true,
+	}
+	return schm
+}
+
+func resourceLoggingProjectExclusionCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	id, exclusion := expandResourceLoggingExclusion(d, "projects", project)
+
+	_, err = config.clientLogging.Projects.Exclusions.Create(id.parent(), exclusion).Do()
+	if err != nil {
+		return err
+	}
+
+	d.SetId(id.canonicalId())
+
+	return resourceLoggingProjectExclusionRead(d, meta)
+}
+
+func resourceLoggingProjectExclusionRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	exclusion, err := config.clientLogging.Projects.Exclusions.Get(d.Id()).Do()
+	if err != nil {
+		return handleNotFoundError(err, d, fmt.Sprintf("Project Logging Exclusion %s", d.Get("name").(string)))
+	}
+
+	flattenResourceLoggingExclusion(d, exclusion)
```

Comment:
You should also set the project here in case it was imported from a project that isn't the provider default (and then consequently, set "project" to computed in the schema)